is-game-over.js

```diff
@@ -0,0 +1,13 @@
+"use strict"
+const _ = require("lodash");
+
+function isGameOver(gameState) {
+    const activePlayer = gameState.isPlayerOne ? gameState.playerOne : gameState.playerTwo;
+    if (activePlayer.winningPieces === 15 && activePlayer.barPieces === 0 && activePlayer.initialPieces === 0) {
```

Comment:
I would love to see  `15` extracted into a constant somewhere instead of being a [magic number](http://c2.com/cgi/wiki?MagicNumber).